corral/vcs/vcs.pony

```diff
@@ -32,17 +31,15 @@ primitive VcsForType
       NoneVcs
     end
 
-
 interface val Vcs
```

Comment:
under the style guide this should be VCS not VCS. same for other uses of VCS.

Reply:
Uck. I see that section in the guide: "Acronyms (HTTP, JSON, XML) in types are uppercase." but I don't like it... This style leads to unreadable identifiers when there are adjacent initialisms, like HTTPURL. The language identifier casing rules should override the English casing of the component words.

BTW, two of the three examples of acronyms are not acronyms: HTTP and XML are not pronounced, so they are just initialisms and not acronyms. We should use "initialism" in the style guide and not "acronym". (and we should fix this casing rule mistake :)

Reply:
The fun thing about group decision standards is that we sometimes take on some things we don't like. I've come to accept the things I don't like about the style guide. I don't really want to reopen the discussion of this point and if changed, introduce breaking changes into the standard library and other projects.

Reply:
OK, done. I mostly don't hate it :)